scripts/core/editor3/components/spellchecker/SpellcheckerContextMenu.tsx

```diff
@@ -66,7 +72,7 @@ export class SpellcheckerContextMenuComponent extends React.Component<IProps> {
                                         }
                                         data-test-id="spellchecker-menu--suggestion"
                                     >
-                                        {messageExists ? message : suggestion.text}
+                                        {suggestion.text}
```

Comment:
Did you check if it works well with `grammalecte`? I think @pablopunk added the ternary to handle a case when `suggestion.text` is whitespace. The message is then was used so the button can be clicked. Can you test if your change works with white-space suggestions?

Reply:
yep, it doesn't look good when the suggestion is whitespace.

![space-suggestion](https://user-images.githubusercontent.com/10600741/64415614-48c36880-d0b3-11e9-9f31-bfc893233836.png)